src/lib/network_pool/transaction_pool.ml

```diff
@@ -2272,124 +2267,6 @@ let%test_module _ =
           assert_pool_txs t (List.drop few_now 1) ;
           Deferred.unit )
 
-    let%test_unit "Expired transactions that are already in the pool are \
```

Comment:
This test never tested the `valid_until` field of a zkApp transaction. Could you please add a test similar to this but testing that expired zkApp transactions (based on the `valid_until` field) are removed from the pool?